addons/traefik/Enable.ps1

```diff
@@ -98,25 +98,55 @@ Write-Log 'Installing external-dns' -Console
 $externalDnsConfig = Get-ExternalDnsConfigDir
 (Invoke-Kubectl -Params 'apply' , '-k', $externalDnsConfig).Output | Write-Log
 
-Write-Log 'Installing Traefik Ingress controller' -Console
-$traefikYamlDir = Get-TraefikYamlDir
+Write-Log "Preparing kusomization with $controlPlaneIp as an external IP for traefik service" -Console
```

Comment:
Typo kusomization -> kustomization.
Rest looks good, I like the approach of patch using kustomize. Should be used in all addons as well.

Reply:
Thanks for your feedback, Mahesh